cpp/wrappers/WKTJsiArrayWrapper.h

```diff
@@ -208,11 +292,17 @@ class JsiArrayWrapper : public JsiHostObject,
 
     auto wrappedArg = JsiWrapper::wrap(runtime, arguments[0], nullptr,
                                        getUseProxiesForUnwrapping());
-    for (size_t i = 0; i < _array.size(); i++) {
+    
+    double fromIndex = 0;
```

Comment:
maybe there is a good reason for all indexes to be doubles, I am just wondering what reason that is? 😄

Reply:
Corrected - should be `size_t` :) Was blinded by the doubles!